platform/viewer/src/index.js

```diff
@@ -35,15 +27,9 @@ const appDefaults = {
 
 if (window) {
   config = window.config || {};
-  config.extensions = [
-    OHIFVTKExtension,
-    OHIFDicomHtmlExtension,
-    OHIFDicomMicroscopyExtension,
-    OHIFDicomPDFExtension,
-  ];
```

Comment:
If you remove these from here, and shift them to `App.js`, then they will be bundled in `@ohif/viewer`. We only want them in the PWA application, so we can keep `@ohif/viewer` lightweight (only cornerstone extension, with others explicitly added).

Reply:
Just moved this code back and made some changes in the props (App.js), what about now?